frontends/election-manager/src/components/export_election_ballot_package_modal_button.tsx

```diff
@@ -157,12 +156,9 @@ export function ExportElectionBallotPackageModalButton(): JSX.Element {
   // Callback to open the file dialog.
   async function saveFileCallback(openDialog: boolean) {
     assert(state.type === 'ArchiveBegin');
-    assert(currentUserSession); // TODO(auth) check proper file permissions
+    // TODO(auth) check proper file permissions
```

Comment:
I wasn't sure what to do with most of these auth todos, so I just left them. But they kind of seem unnecessary since we don't have permissions for specific files/actions